packaging/linux/systemd/kbfs.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Keybase Filesystem service
+Wants=keybase.service
+
+[Service]
+# "notify" means we promise to call SdNotify() at the end of startup.
+Type=notify
+# Forcibly unmount /keybase in case there's anything there. The "-" prefix
+# means that error codes from this command are ignored. Without this line,
+# `systemctl --user restart kbfs.service` will hit mount failures if there
+# are any running shells cd'd into a Keybase folder.
+ExecStartPre=-/bin/sh -c 'fusermount -uz "$(keybase config get mountdir|sed s/\"//g)"'
```

Comment:
Thanks for cleaning up the quoting here. We had definitely broken this command. That said, I think we still need to tweak this line. I get a parsing failure here, and I think it's because systemd's interpretation of single quotes is different from sh's. We probably need to double that backslash?